### ISSUE TITLE
Nocash-sym Error

### ISSUE THREAD

**Issue by Fechdog** [LeonarthCG/FFTA_Engine_Hacks#5]
I'm new to this kind of tool for gba where i need multiple programs to download. Whenever I create a new build, I keep getting this error: https://i.imgur.com/OKWyh24.png

I have absolutely no idea what it means and I would like to know if it will be possible to use this for a randomizer with this error.

PS: Even though the event file has the intro off, it's still playing the intro despite quick start not being commented out.

**Comment by LeonarthCG** [comment]
You are using an older core for EA, which does not support generating symbol information for no$, this doesn't really matter if you don't plan on using the debugger, you can either remove the flag from the .cmd file or get the new core here: https://github.com/FireEmblemUniverse/ColorzCore/blob/master/ColorzCore/bin/Release/ColorzCore.exe
Then you can replace the old one inside the Event Assembler folder.

Similarly, you did not download ups.exe, but this only matters if you wanted to automatically generate patch files.

As for the options, I'm having trouble trying to understand what you meant.
Are you talking about the intro cutscene that appears before the press start screen, and thus the skipIntroCutscene option, or are you talking about quickStart and the first few in-game battles?
These two options are completely unrelated.
Either way, since the building process failed, no changes were applied to the ROM at all, like the message says, so everything should appear to be completely vanilla.

**Comment by Fechdog** [comment]
I would like to focus more on the lower part on that image saying 'FFTA_hack.ups' is not recognized as a command. I downloaded the ColorzCore and replace the regular core in Event Assembler _and_ removed the flag from the .cmd file and I'm still getting an error about the ups file. I did download it as the page suggested and put it in the root directory. I'm guessing this is preventing the writing to the new rom.

**Comment by LeonarthCG** [comment]
It's not preventing the ROM from being modified at all. 
ups.exe generates .ups files, which can be used to apply the changes to a clean ROM, but the output ROM already has the changes applied to it either way, the patch is only for distribution.

ups.exe is a completely different program so nothing that you did or could do to Event Assembler or the flags is going to make any difference on that second part.

I'd also like to take this opportunity to warn you about automatic patching: some emulators have an option to apply patches to ROMs with matching names automatically. When this is done with a ROM that already has those changes, the changes are undone and you end up with a vanilla ROM.

Either way, just to make sure, did you extract ups.exe to the root folder, or did you extract the ups folder, with the ups.exe file inside of it, to the root folder?
The instructions say to do the latter.

To summarize: ups.exe is only called after the ROM is all done in order to create a patch, so if you don't care about that then it has no impact and you can either completely ignore the error or delete the related lines.

**Comment by Fechdog** [comment]
I extracted the ups.exe file into the directory with the 'MAKE_HACK.cmd' file.

**Comment by LeonarthCG** [comment]
The instructions say to extract the ups folder, with ups.exe inside of it, to the root folder, so you would have a ups folder in the root folder and ups.exe inside of that.

**Comment by Fechdog** [comment]
I got it working! Thanks so much for the help and sorry for the trouble!
